### DISCUSSION TITLE
Git push does not finish!

### DISCUSSION THREAD

**Discussion by ArnelaKapllani** [neuefische/questions#643] [Web] [answered]
I tried to push from local repository to remote but it does not work. Git push seems to be frozen. 

<img width="511" alt="Screenshot 2023-07-18 at 15 50 37" src="https://github.com/neuefische/questions/assets/130066692/36e208ce-54cd-4e5e-ac61-7852d137d2b5">

Even though I waited many minutes, it still doesn't finish. I didn't do any new actions or any tricks, I just worked normally as usual.

Thank you in advance 🙏🙂

**Comment by marcbendixen** [comment, marked as answer]
Hey @ArnelaKapllani 👋 

some ideas:
- are your changes/commits visible on GitHub? [Link to history of branch feature/shoppingCart](https://github.com/ArnelaKapllani/capstone-arnela/commits/feature/shoppingCart)
- try to completely quit terminal/VSCode and push again
- check with `git remote -v` if the correct remote url is configured in your project

**Reply by ArnelaKapllani** [reply]
Thank you @marcbendixen 🙏 A restart of VSCode helped 🙂
